.github/workflows/e2e.yml

```diff
@@ -20,6 +20,7 @@ on:
       - master
       - release/*
       - e2e-test
+      - monitoring_tests_to_e2e
```

Comment:
Dan told me to add this branch to the list so it would run on this PR but I'm not sure if I did it correctly.. @Daniel-Sanche could you advise here :)

Reply:
This addition is necessary to see the tests running in this PR, since usually they'd only run after each merge

I'd recommend trying to remove this after all approvals and before merging, but it's not a big deal if it accidentally sticks around